tests/test_apps/helloworld/hello.py

```diff
@@ -6,3 +6,6 @@
 @app.route("/")
 def hello():
     return "Hello World!"
+
+if __name__=="__main__:
```

Comment:
The formatting issues aside, this isn't even valid code (missing `"`) - so not only does this PR not improve anything, it is actually broken and indicates that you never bothered to test the change you proposed. Please stop doing that, it's just unnecessary noise for any project receiving such PRs.